CHANGELOG.md

```diff
@@ -1,5 +1,310 @@
 # Themis ChangeLog
 
+## [0.12.0](https://github.com/cossacklabs/themis/releases/tag/0.12.0), September 23rd 2019
+
+**TL;DR:**
+Added WasmThemis to support WebAssembly,
+improved packaging and platform compatibility,
+additional safety checks and tricky bug fixes.
+
+**Breaking changes:**
+
+- Default installation path has been changed from `/usr` to `/usr/local`.
+- Some API of GoThemis have been renamed (old API is considered deprecated).
+- Deprecated `rubythemis` gem has been completely removed.
+
+_Code:_
+
+- **Core**
+
+  - **Soter** (low-level security core used by Themis)
+
+    - New function: `soter_wipe()`.
+      It can be used to securely wipe sensitive data from memory after it's no longer necessary.
+      All Themis cryptosystems now make use of this new API.
+      ([#488](https://github.com/cossacklabs/themis/pull/488))
+
+    - Non-cryptographically secure PRNG could be used if the backend does not provide a cryptographically secure one.
```

Comment:
```suggestion
    - Improved usage and error handling of OpenSSL/BoringSSL PRNGs.
```

actually, i don't see how this commit solves "Non-cryptographically secure PRNG could be used if ..."

Reply:
`RAND_bytes()` could return negative value to indicate that CSPRNG is not available and leave the buffer in undefined state (no idea what OpenSSL actually does there). We were treating this error code as success and went on with our business, assuming that we got the random bytes we needed. With this change we correctly report an error and do not assume that ‘random crap found on the call stack’ is a PRNG in any capacity.

Anyway, I like your wording better for being concise and significant.